Dockerfile

```diff
@@ -17,13 +17,12 @@ FROM openjdk:8 as final
 
 WORKDIR /app
 
-ENV HEALTHCHECKURL http://localhost:8080/actuator/health
 ARG JAVA_OPTS
 
 COPY ./a2d2-api/target/a2d2-api.war /app/a2d2-api.war
 COPY --from=maven /usr/src/client_repo /root/.m2/repository
 COPY --from=maven /usr/src/services /app/services
 
-HEALTHCHECK --interval=60s --timeout=30s --start-period=30s --retries=3 CMD curl -f $HEALTHCHECKURL 2>&1 | grep UP || exit 1
```

Comment:
Why are you removing the health check? These were added to ensure that docker could determine if the service is healthy.

Reply:
Hank, removed the health check URL which is on 8080, as we are specifying the port and health check on runtime.

Reply:
That doesn't explain why you are removing it. It's defined on port 8080, which is one of the ports allowed for this container. The team went through all services and added these health checks in the past.

Reply:
I have reverted changes for healthcheck, Please review it now.